TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/ContentCache.php

```diff
@@ -237,11 +243,14 @@ public function getCachedSegment($uncachedCommandCallback, $typoScriptPath, $cac
     public function replaceCachePlaceholders(&$content, $addCacheSegmentMarkersToPlaceholders)
     {
         $cache = $this->cache;
+        $debugger = $this->debugger;
```

Comment:
@100hz kann man das nicht besser direkt auf `$this->debugger` zugreifen anstatt eine lokale Variable dafür zu verwenden? Ich bin mir nicht 100% sicher, denke aber, dass Flow das, sobald man darauf zugreift, materialisiert, was u.U. aber gar nicht sein muss, wenn die untere Condition, also `if ($entry !== false)` false ist. So bleibt man lazy und der Code ist einfacher zu lesen (in diesem Falle vermutlich egal, aber oft werden Funktionen mit der Zeit größer und da ist das hinderlich, wenn man die Variablen zurück verfolgen muss.